src/index.html

```diff
@@ -11,7 +11,205 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      id="our-form"
+    >
+      <fieldset class="form-box">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label for="surname">Surname:</label> 
+
+          <input 
+              type="text" 
+              name="surname" 
+              id="surname"
+              autocomplete="off" 
+              required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="name">Name:</label>
+
+          <input 
+              type="text" 
+              name="name" 
+              id="name"
+              autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+
+          <input 
+              type="number"
+              name="age"
+              id="age"
+              pattern="\d+"
+              value="12"
+              min="1"
+              max="100"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="dateBirth">Full date of birth:</label>
+
+          <input 
+              type="date"
+              name="dateBirth"
+              id="dateBirth"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="accept">I accept the term of the agreement</label>
+
+          <input 
+              type="checkbox"
+              name="acceptAgreement"
+              id="accept"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-box">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+
+          <input 
+              type="email"
+              placeholder="email@example.com"
+              name="email"
+              id="email"
+              autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="password">Password: </label>
+
+          <input 
+              type="password"
+              name="password"
+              id="password"
+              minlength="5"
+              maxlength="15"
+              autocomplete="off"
+              required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-box">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          <label for="cats">Do you love cats?</label>
+
+          <input 
+              type="radio" 
+              name="cats"  
+              value="Yes"
+          >
+
+          <label for="cats">Yes</label> 
```

Comment:
One last thing. Fix labels for radio buttons so they are clicable and focuses the radio button. Add propper **ids** so label **for** is working properly
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label